benchmark_schema.py

```diff
@@ -0,0 +1,49 @@
+from sqlalchemy import Column, DateTime, Float, Integer, String
+from sqlalchemy.orm import declarative_base
+
+Base = declarative_base()
+
+
+class TestRun(Base):
+    __tablename__ = "test_run"
+
+    # unique run ID
+    id = Column(Integer, primary_key=True)
+
+    # pytest data
+    session_id = Column(String, nullable=False)
+    name = Column(String, nullable=False)
+    originalname = Column(String, nullable=False)
+    path = Column(String, nullable=True)
+    setup_outcome = Column(String, nullable=True)
+    call_outcome = Column(String, nullable=True)
+    teardown_outcome = Column(String, nullable=True)
+
+    # Runtime data
+    coiled_runtime_version = Column(String, nullable=True)
+    coiled_software_name = Column(String, nullable=True)
+    dask_version = Column(String, nullable=True)
+    python_version = Column(String, nullable=True)
+    platform = Column(String, nullable=True)
+
+    # CI runner data
+    ci_run_url = Column(String, nullable=True)
+
+    # Wall clock data
+    start = Column(DateTime, nullable=True)
+    end = Column(DateTime, nullable=True)
+    duration = Column(Float, nullable=True)
+
+    # Memory data
+    average_memory = Column(Float, nullable=True)
+    peak_memory = Column(Float, nullable=True)
+
+    # Timing data
+    compute_time = Column(Float, nullable=True)
+    disk_spill_time = Column(Float, nullable=True)
+    serializing_time = Column(Float, nullable=True)
+    transfer_time = Column(Float, nullable=True)
```

Comment:
I could be missing something, but are we collecting this information?

Reply:
No, we are not. These, as well as URLs for cluster dumps and performance reports, are not currently being collected, but I think we'll want to collect them soon, and I wanted to save us a database migration.

Reply:
How expensive are migrations? If they're cheap, I'd prefer to have the current database schema reflect information that's actually being gathered. Otherwise, could you add a comment here mentioning this information isn't yet collected?

Reply:
They are more expensive from a nuisance perspective than a disk storage or time perspective. They involve a bit of work, potentially from someone who is unfamiliar with them, and can be easy to get wrong.

I'd prefer to add a comment, since I think I'd like to add these pretty soon, if that's okay

Reply:
Sounds good, I'll defer to your judgement here. Thanks for the additional context